osgp/shared/osgp-throttling-client/src/main/java/org/opensmartgridplatform/throttling/ThrottlingClient.java

```diff
@@ -108,7 +128,14 @@ private void registerThrottlingConfig() {
     LOGGER.info("Registered {}", this.throttlingConfig);
   }
 
-  private void registerClient() {
+  /*
+   * registerThrottlingConfig should be synchronized, to make sure there is only
```

Comment:
Copy paste issue: registerThrottlingConfig should be registerClient here.

Reply:
changed